### ISSUE TITLE
[destination-databend] Create new connector error `[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)`

### ISSUE THREAD

**Issue by kulame** [airbytehq/airbyte#34473]
### Connector Name

Databend

### Connector Version

0.1.2

### What step the error happened?

Configuring a new connector

### Relevant information

Configuration check failed

show https connect error 

### Relevant log output

```shell
Configuration check failed

An exception occurred: SSLError(MaxRetryError("HTTPSConnectionPool(host='host.docker.internal', port=8500): Max retries exceeded with url: /v1/query/ (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)')))"))
```

### Contribute

- [ ] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
There is a PR to fix the issue: https://github.com/airbytehq/airbyte/pull/34481

**Comment by hantmac** [comment]
> There is a PR to fix the issue: #34481

@marcosmarxm Hi, this pr will fix this issue, when will we merge it? Or I make a new pr to fix it?
